### ISSUE TITLE
[TwigBridge] Bootstrap 3 horizontal layout not renders correctly the checkbox

### ISSUE THREAD

**Issue by antalaron** [symfony/symfony#24702]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.4.0-BETA1

In 3.4.0-BETA and probably in 4.0.0-BETA, because of the refactorization, and extension of Bootstrap form themes in #21751 the checkbox render is failing for Bootstrap 3.

3.3.10 (good):
![kepernyofoto 2017-10-26 - 21 58 37](https://user-images.githubusercontent.com/9386504/32074191-efc33f6c-ba98-11e7-8929-cf816dd9d2c6.png)

3.4.0-BETA (bad, the placeholder for the label is missing):
![kepernyofoto 2017-10-26 - 21 58 06](https://user-images.githubusercontent.com/9386504/32074202-f51d8170-ba98-11e7-88a5-c1f22295d912.png)

Here is a sample reproduction with 2 branches (stable, beta):
https://github.com/antalaron/symfony-issue-checkbox

I checked, if re-include the original code for checkboxes to `bootstrap_3_horizontal_layout.html.twig`, it's ok:

```twig
{% block checkbox_row -%}
    {{- block('checkbox_radio_row') -}}
{%- endblock checkbox_row %}

{% block checkbox_radio_row -%}
{% spaceless %}
    <div class="form-group{% if not valid %} has-error{% endif %}">
        <div class="{{ block('form_label_class') }}"></div>
        <div class="{{ block('form_group_class') }}">
            {{ form_widget(form) }}
            {{ form_errors(form) }}
        </div>
    </div>
{% endspaceless %}
{%- endblock checkbox_radio_row %}
```

But I don't know if it breaks anything else.

**Comment by xabbuh** [comment]
should be fixed by #24711

**Comment by antalaron** [comment]
Yes. I can confirm that.
